channels/test/liveserver.py

```diff
@@ -113,6 +113,12 @@ def run(self):
 
 
 class ChannelLiveServerTestCase(TransactionTestCase):
+    """
+    Does basically the same as TransactionTestCase but also launches a
+    live Daphne server and Channels worker in a separate processes so
+    that the tests may use another testing framework, such as Selenium
+    for example, instead of the built-in dummy client.
```

Comment:
I think that should read "... test framework, such as Selenium, ..." (no -ing; no for example)

Just checking that we're PEP20 compliant! ("[Readability counts.](https://www.python.org/dev/peps/pep-0020/)")  :1st_place_medal: :smiley: